packages/components/src/constants.js

```diff
@@ -3,6 +3,7 @@ export const CIRCULAR_PROGRESS_SIZE = {
 	small: 'small',
 	default: 'default',
 	large: 'large',
+	larger: 'larger',
```

Comment:
Should be named `xlarge` to allow for even bigger sizes.

Reply:
fixed